flink-cdc-e2e-tests/flink-cdc-pipeline-e2e-tests/src/test/java/org/apache/flink/cdc/pipeline/tests/utils/PipelineTestEnvironment.java

```diff
@@ -106,7 +106,7 @@ public static List<String> getFlinkVersion() {
         if (flinkVersion != null) {
             return Collections.singletonList(flinkVersion);
         } else {
-            return Arrays.asList("1.19.1", "1.20.0");
+            return Arrays.asList("1.18.1", "1.19.1", "1.20.0");
```

Comment:
Do we need to add previous versions into CI matrix, too?

Reply:
done